products/health-care/digital-health-modernization/engineering/mhv-downtime-notification-discovery.md

```diff
@@ -0,0 +1,52 @@
+# MHV on VA.gov downtime notifications
+
+Created: 2023-12-21
```

Comment:
This document read more like notes. Perhaps add a short introduction describing the purpose of this document. I read it and was not sure what to do with it.

Reply:
I feel the same way as Carlos. This doc reads as notes that could fuel a discussion of recommendations. It's a useful input to that work.

For example: below it's noted that the Appts team has used a `render` prop to customize the UX of the downtimeNotification component, rather than used it out-of-the-box. I'm glad you found that out. But now our team needs to understand the UX of that experience, and take it into account in our UX recommendation. Can we get screenshots?

Reply:
Yes, they are notes from downtime discovery! I can add an intro saying that and adding some recommendations. Was hoping to generate discussion based on what's there, but it sounds like a read out might generate more useful discussion.

@wesrowe I don't have screenshots of the Appointments and don't know about getting them, but will look into it

Reply:
Added a document purpose statement and a *Recommendations* section. And I think those may answer https://github.com/department-of-veterans-affairs/va.gov-team/issues/70798#issuecomment-1876136559 but please let me know what is still confusing

Reply:
@wesrowe Added screenshots from Appointments/VAOS